zips/draft-coinholder-directed-grants.md

```diff
@@ -0,0 +1,192 @@
+
+---
+
+### 3. Participation and Voting Thresholds
+
+* Participation threshold: ≥ 420,000 ZEC voted (≈ 2% of total supply).  
+* Approval criterion: simple majority of “Yes” votes.  
+* Each proposal vote is independent; coins used in one vote MAY be reused in another.  
+
+---
+
+### 4. Voting Process
+
+The Coinholder Grants Program supports two voting methods: **Shielded Voting** and **Transparent Voting**.  
+
+Both MUST produce verifiable and auditable results administered by the Administrator or its delegate.  
+
+#### 4.1 Shielded Voting Process
+
+Shielded voting is conducted through the dedicated Coin Voting 2.0 application that uses Orchard addresses and the Halo2 proof system to verify balances and prevent double voting.  
+
+1. **Registration Period** – The Administrator or its delegate MUST define a registration window not shorter than 10 days, specified by block heights. Only Orchard notes created or refreshed within this window are eligible for voting.  
+2. **Pre-Vote Safeguards** – Participants SHOULD move funds out of the wallet used for voting *before* entering their seed phrase into the Coin Voting application to reduce risk if the third-party app is compromised.  
+3. **Voting Phase** – After registration closes, eligible coinholders vote using their Orchard funds via the Coin Voting application. Each ZEC held corresponds to 1,000 vote units.  
+4. **Vote Submission** – Votes are submitted to a dedicated voting blockchain operated by independent Voting Authorities running a Byzantine Fault Tolerant (BFT) consensus network (e.g., CometBFT).  
+5. **Tally and Audit** – At the end of the voting period, the Administrator or its delegate collects finalized results from the Voting Authorities. Auditors MAY independently verify the tally using the public viewing key and revealed election seed.  
+6. **Security Assumptions** – The system MUST include ≥ 4 Voting Authorities to ensure liveness and integrity; so long as ≥ ⅔ are honest, the vote cannot be manipulated.  
+
+#### 4.2 Transparent Voting Process
+
+Transparent voting MAY be used alongside shielded voting to expand participation and maintain verifiability.  
+
+1. Voters create a signed message containing their vote choice and transparent address.  
+2. The signed message and a payment of ≥ 1 ZEC MUST be sent to a designated Orchard address controlled by the Administrator or its delegate.  
+3. The Administrator or its delegate publishes the corresponding viewing key so anyone can verify vote contents and tally independently.  
+4. Votes with payments < 1 ZEC MUST be ignored to mitigate Sybil attacks.  
+5. Funds MUST remain immobile until the voting period concludes to prevent vote tampering.  
+
+Both methods MUST produce publicly auditable vote records and a final summary published by the Administrator or its delegate within 7 days of vote conclusion.  
```

Comment:
The Administrator must produce a publicly auditable vote record; whether the method produces an auditable vote record is something that the reviewers/ZIP editors will check.

The need to produce a publicly auditable vote record is part of the requirements.